src/components/ImportWizard/SourceClusterProjectStep.tsx

```diff
@@ -64,43 +66,88 @@ export const SourceClusterProjectStep: React.FunctionComponent = () => {
       ...(valid ? { validated: 'success' } : {}),
     };
-    const formGroupProps: Pick<FormGroupProps, 'validated' | 'helperText'> = {
+    const formGroupProps: Pick<FormGroupProps, 'validated' | 'helperText' | 'labelIcon'> = {
       ...inputProps,
       helperText: validating ? 'Validating...' : helperText,
+      labelIcon: labelIcon,
     };
     return { inputProps, formGroupProps };
   };
 
-  const apiUrlFieldProps = getAsyncValidationFieldProps(
-    credentialsValidating,
-    credentialsAreValid,
-    <div className={formStyles.formHelperText}>
-      API URL of the source cluster, e.g. <code>https://api.example.cluster:6443</code>
-    </div>,
-  );
+  const apiUrlFieldProps = getAsyncValidationFieldProps({
+    validating: credentialsValidating,
+    valid: credentialsAreValid,
+    labelIcon: (
+      <Popover
+        headerContent={`API URL of the source cluster`}
+        bodyContent={
+          <span>
+            e.g. <code>https://api.example.cluster:6443</code>
+          </span>
+        }
+      >
+        <button
+          type="button"
+          aria-label="More info for api url field"
+          onClick={(e) => e.preventDefault()}
+          aria-describedby="api-url"
+          className="pf-c-form__group-label-help"
+        >
+          <HelpIcon noVerticalAlign />
+        </button>
+      </Popover>
+    ),
+    helperText: <>&nbsp;</>,
+  });
 
-  const sourceTokenFieldProps = getAsyncValidationFieldProps(
-    credentialsValidating,
-    credentialsAreValid,
-    <div className={formStyles.formHelperText}>
-      OAuth token of the source cluster. Can be found via <code>oc whoami -t</code>
-    </div>,
-  );
+  const sourceTokenFieldProps = getAsyncValidationFieldProps({
+    validating: credentialsValidating,
+    valid: credentialsAreValid,
+    labelIcon: (
+      <Popover
+        headerContent={`OAuth token of the source cluster`}
+        bodyContent={
+          <span>
+            Can be found via <code>oc whoami -t</code>
+          </span>
+        }
+      >
+        <button
+          type="button"
+          aria-label="More info for oauth token field"
+          onClick={(e) => e.preventDefault()}
+          aria-describedby="token"
+          className="pf-c-form__group-label-help"
+        >
+          <HelpIcon noVerticalAlign />
+        </button>
+      </Popover>
+    ),
+    helperText: <>&nbsp;</>,
+  });
 
-  const sourceNamespaceFieldProps = getAsyncValidationFieldProps(
-    validateSourceNamespaceQuery.isLoading,
-    validateSourceNamespaceQuery.data?.data.kind === 'Namespace',
-    <div className={formStyles.formHelperText}>Name of the project to be migrated</div>,
-  );
+  const sourceNamespaceFieldProps = getAsyncValidationFieldProps({
+    validating: validateSourceNamespaceQuery.isLoading,
+    valid: validateSourceNamespaceQuery.data?.data.kind === 'Namespace',
+    helperText: <div className={formStyles.formHelperText}>Name of the project to be imported</div>,
+  });
```

Comment:
If you wanted to add a Popover for this option too, similar to the `oc whoami -t` , `oc project` will tell you the project currently in use. From the `oc` help text:

```
Examples:
  # Switch to 'myapp' project
  oc project myapp

  # Display the project currently in use
  oc project
```

Looking at the screenshot shared it wasn't clear how easy it would be for the user to get this help.

Reply:
<img width="917" alt="Screen Shot 2022-04-18 at 4 38 03 PM" src="https://user-images.githubusercontent.com/5942899/163874329-f891c73a-1efc-4b91-9472-9bac45e4582f.png">

Good call! Styling seems to still be getting tweaked in the host app, so the `<code>` snippet has interesting styling - can fix it up here if need be but I think this does the main thing you're after. Lemme know if you see anything else!